include/deal.II/lac/la_vector.templates.h

```diff
@@ -560,6 +560,25 @@ namespace LinearAlgebra
 
 
 
+  template <typename Number>
+  void
+  Vector<Number>::print_numpy_array(std::ostream &     out,
+                                    const unsigned int precision) const
+  {
+    AssertThrow(out, ExcIO());
+
+    out.precision(precision);
```

Comment:
We should probably restore the previous precision after calling this function.

Reply:
this can be done using ``boost::io::ios_flags_saver`` for example.